drip/models.py

```diff
@@ -186,17 +190,17 @@ def annotated_field_name(self) -> str:
 
         return field_name
 
-    def apply_any_annotation(self, qs: Optional[QuerySet]) -> Optional[QuerySet]:
+    def apply_any_annotation(self, qs: QuerySet) -> QuerySet:
         """
         Returns qs annotated with Count over this field's name.
         """
-        if self.field_name.endswith("__count") and qs is not None:
```

Comment:
Why did you remove `and qs is not None`?

Reply:
I added it in another PR, but now it is not needed because this function should not get a None QuerySet.